compiler/test/dotty/tools/repl/TabcompleteTests.scala

```diff
@@ -29,6 +29,13 @@ class TabcompleteTests extends ReplTest {
       comp.suggestions.sorted == List("slice", "sliding"))
   }
 
+  @Test def tabCompleteModule: Unit =
+    fromInitialState{ implicit s =>
+      val comp = tabComplete("scala.Pred").suggestions
+    assertTrue(s""""Expected single element "Predef" got: ${comp}""",
```

Comment:
Weird indentation

Reply:
You don't need braces after `$` when you only have an identifier: `$comp`

Reply:
Also, I actually think it is better to use: `assertEquals(comp.suggestions, List("Predef"))` than `assertTrue`. The message will be automatically generated.